src/test/java/org/jenkinsci/plugins/workflow/support/pickles/ExecutorPickleTest.java

```diff
@@ -38,12 +42,15 @@
 import org.junit.Rule;
 import org.junit.runners.model.Statement;
 import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.MockAuthorizationStrategy;
 import org.jvnet.hudson.test.RestartableJenkinsRule;
 
 public class ExecutorPickleTest {
 
     @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();
     @Rule public RestartableJenkinsRule r = new RestartableJenkinsRule();
+    //@Rule public LoggerRule logging = new LoggerRule().record(Queue.class, Level.FINE);
```

Comment:
Useful with https://github.com/jenkinsci/jenkins/pull/2791, and triggers further exceptions fixed in https://github.com/jenkinsci/jenkins/pull/2792.